server/resources/[VORP]/[homestead_essentials]/homestead_jobs/server/functions.lua

```diff
@@ -0,0 +1,21 @@
+
+--- @param source integer
+--- @param job string
+--- @param jobgrade integer
+function validateJob(job, jobgrade)
+    local jobsData = LoadResourceFile(GetCurrentResourceName(), "shared/jobs.json")
+    jobsData = json.decode(jobsData)
+    local exists = false
+    for job, grades in pairs(jobsData.Jobs) do
```

Comment:
if we remove "Jobs" parent key we will be able to do something like

return jobsData[job] and jobsData[job][jobGrade] something like that

@BuffsOverNexus can you throw second opinion if possible?